src/main/java/com/conveyal/r5/analyst/cluster/AccessGridWriter.java

```diff
@@ -0,0 +1,154 @@
+package com.conveyal.r5.analyst.cluster;
+
+import com.conveyal.r5.analyst.LittleEndianIntOutputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.nio.ByteBuffer;
+
+/**
+ * Random-access access grid writer - write arrays into offsets in an access grid format file (3-d array)
+ *  * Access grids look like this:
+ * Header (ASCII text "ACCESSGR") (note that this header is eight bytes, so the full grid can be mapped into a
```

Comment:
I just meant that the term "header" (used in the javadoc) usually refers to the entire block of meta-info that precedes the main repeating block of data in a file. So those letters ACCESSGR are just the first element in the header (as I see it).